R/scripts/setup_initial_nonUS_data.R

```diff
@@ -0,0 +1,179 @@
+
```

Comment:
seeing some hard coded values here.  wondering if this file should either be:
 - integrated with the config
 - made into an Rmd workspace with parameters
 - made into a script with command line arguments
 - not committed
might be none of these, just not sure

Reply:
ok now its default is command line.